beacon-chain/cache/depositsnapshot/graph.go

```diff
@@ -0,0 +1,59 @@
+package depositsnapshot
+
+import (
+	"bufio"
+	"fmt"
+	"os"
+)
+
+func printTree(tree MerkleTreeNode) error {
```

Comment:
Used this function for debugging, not entirely sure what to do with it otherwise.